CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# How to contribute to FreeOrion
+
+As open source project FreeOrion both relies on contributions from both long
+time developers and one time occasion contributors.  This guide should help you
```

Comment:
I find adding two spaces after more readable for for monospace fonts in terminal editors so I adopted that style for code comments and plain text.  The rendered HTML doesn't matter as the HTML specification requires to collapse any number of whitespace into a single space character.

See also [english spacing](https://en.wikipedia.org/wiki/Sentence_spacing) for more details.